basic_datastructure/basic_array/array.PHP

```diff
@@ -0,0 +1,152 @@
+<?php 
+    //------------------------------HOMEWORK 1----------------------------------
+
+    class SimpleArray {
+        /** @var array */
+        public $arr;
+
+        /**
+        * constructor
+        * @param array $arr
+        */
+        public function __construct($arr)
+        {
+            $this->arr = $arr;
+        }
+
+        /**
+        * calculate the sum of elements in array
+        * @return int
+        */
+        public function sum() 
+        {
+            return array_sum($this->arr);
+        }
+
+        /**
+        * calculate the average of elements in array
+        * @return float
+        */
+        public function average() 
+        {
+            return $this->sum() / count($this->arr);
+        }
+
+        /**
+        * calculate the min of elements in array
+        * @return int
+        */
+        public function min() 
+        {
+            $cpyArr = $this->arr;
+            sort($cpyArr);
+            return $cpyArr[0];
+        }
+
+        /**
+        * calculate the max of elements in array
+        * @return int
+        */
+        public function max() 
+        { 
+            $cpyArr = $this->arr;
+            rsort($cpyArr);
+            return $cpyArr[0];
+        }
+
+        /**
+        * print the average, sum, min and max of elements in array
+        * @return void
+        */
+        public function output() 

```

Comment:
Given an array with N elements, calculate the average, sum, min and max of elements in array
-> Why do we need many function in here?
average -> you call sum method and count method -> BigO: log(2n).
sum -> BigO: log(n).
min -> BigO: nlog(n).
max-> BigO: nlog(n).


Please fix by 1 method and 1 loop. Expected: BigO: log(n).

Reply:
Thanks for your comment. Do you mean i should :
- sort array -> min , max -> BigO: nlog(n).
- sum -> call array_sum method -> BigO: log(n).
- average -> call count method -> BigO: log(n).

Reply:
@Khiemnx Yes. We don't need use collectionUtils method for this output method.

Reply:
i have fixed this problem.
Thanks for your comment